### PR TITLE
dev-tex/cpp2latex: HOMEPAGE and SRC_URI moved; Rewrite for EAPI=6

### PR DESCRIPTION
**Pull request by jonasstein**
fixes https://bugs.gentoo.org/show_bug.cgi?id=533844

### CODE REVIEW

**Comment by jonasstein** [comment]
fixes https://bugs.gentoo.org/show_bug.cgi?id=533844

File: dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
 
-# although it makes sense to have tex installed, it is
-# neither a compile or runtime dependency
+DEPEND=""
+RDEPEND=""
+
+S="${WORKDIR}/${P}/cpp2latex"
+
+src_prepare(){
+	echo `ls -al main.cpp`
```
Comment (floppym) [inline dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild]: This looks like some leftover debug code. Probably should remove it.

File: dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
 KEYWORDS="amd64 ppc sparc x86"
 
-# although it makes sense to have tex installed, it is
-# neither a compile or runtime dependency
+DEPEND=""
+RDEPEND=""
+
+S="${WORKDIR}/${P}/cpp2latex"
+
+src_prepare(){
```
Comment (floppym) [inline dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild]: Convert the patches to PATCHES array and drop `src_prepare`.

**Comment by jonasstein** [comment]
going to improve this: 
Todo: remove debug line, use eapply and patches

File: dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild [hunk 3]
```diff
@@ -1,31 +1,33 @@
-# Copyright 1999-2008 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
+EAPI=6
+
 inherit eutils
```
Comment (floppym) [inline dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild]: Drop this once you get rid of `epatch`.

File: dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild (hunk 2 above)
Comment (hartwork) [inline dev-tex/cpp2latex/cpp2latex-2.3-r1.ebuild]: Also `cpp2latex` -> `${PN}`?

**Pull request closed without merging**